Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -259,66 +268,29 @@ open class AVFoundationPlayback: Playback {
         return isDvrAvailable && isEpochInsideDVRWindow(liveStartTime)
     }
 
-    private func createPlayerInstance(with item: AVPlayerItem) {
-        player = shouldLoop ? AVQueuePlayer(playerItem: item): AVPlayer(playerItem: item)
-        if let player = player as? AVQueuePlayer {
-            playerLooper = AVPlayerLooper(player: player, templateItem: item)
+    private func createAVPlayer(with asset: AVAsset? = nil, shouldLoop: Bool) -> AVPlayer {
```

Comment:
```suggestion
    private func createAVPlayer(with asset: AVAsset? = nil) -> AVPlayer {
```
`shouldLoop` is a property that checks a specific key in the `options` (another property), so we don't need it as a parameter.

Reply:
Actually, my initial idea was to remove this property, I don't think it makes sense to keep it as it is used only once. And in doing so (af5c2bedf999e5b6ce6140f4e56c1521de0fe935), as the option is not a Bool but an Any?, I chose to pass `shouldLoop` as a parameter to prevent the function from handling this.